src/app/features/todos/todos.service.ts

```diff
@@ -1,36 +1,15 @@
 import { Injectable } from '@angular/core'
-import { AngularFirestore } from '@angular/fire/firestore'
-import { collectionData } from 'rxfire/firestore'
-import { from, Observable, of } from 'rxjs'
-import { delay, switchMap, take } from 'rxjs/operators'
+import { AngularIdb, AngularIdbService } from '../../angular-idb'
+import { Observable } from 'rxjs'
 import { Todo } from './models/todo'
 
 @Injectable()
```

Comment:
question (non-blocking): is there any reason why we're not making these providedIn: root?

Reply:
Nope, not really. It's a left-over from how I structured it earlier with a separate data-access module, mimicking an Nx structure.